ath79: phy-ar7200-usb: do not print error on defered init

jneuhauser opened the pull request:
This is only a cosmetic correction, as the driver works as expected.
However, the error message confuses users about a missing reset definition.

On a deferd init we don´t see the following error message now:
[    0.078292] ar7200-usb-phy usb-phy: phy reset is missing

Signed-off-by: Johann Neuhauser <johann@it-neuhauser.de>

jneuhauser committed:
ath79: phy-ar7200-usb: do not print error on defered init

This is only a cosmetic correction, as the driver works as expected.
However, the error message confuses users about a missing reset definition.

On a defered init we don't see the following error message now:
[    0.078292] ar7200-usb-phy usb-phy: phy reset is missing

Tested-by: Lech Perczak <lech.perczak@gmail.com>
Signed-off-by: Johann Neuhauser <johann@it-neuhauser.de>

Leo-PL commented:
I had a chance to test this backported to 19.07 on my TL-WDR4300.
You could add my Tested-by tag if you'd like.

adrianschmutzler commented:
Spelling "On a deferd init" -> "On a defered init"
"don´t" -> "don't"

Consider adding the Tested-by and sending this to the devel list to speed it up ...

jneuhauser commented:
Patched also 5.4 patch, corrected the spelling mistakes, added Tested-by @Leo-PL and sent the patch to the mailing list for wider audience.

http://lists.infradead.org/pipermail/openwrt-devel/2020-April/023139.html

jneuhauser commented:
No one is reacting on the mailing list...
So reopen this PR.

jneuhauser commented:
It's merged now, so close this pr.
